### PR TITLE
Switching Flipper over to using REDIS

### PR DESCRIPTION
**Pull request by darkbushido**
## Description of change

Switching Flipper to use the common Redis cache instead of the thread-local memory cache

added a ActiveSupport::Notifications to record the cache hits/misses

## Original issue(s)
department-of-veterans-affairs/va.gov-team#26500

### CODE REVIEW

**Review by dillo — DISMISSED** [review]
Simple solution to a complex problem 👍

**Review by rileyanderson — DISMISSED** [review]

File: config/initializers/flipper.rb [hunk 1]
```diff
@@ -14,11 +14,11 @@
   Flipper.configure do |config|
     config.default do
       activerecord_adapter = Flipper::Adapters::ActiveRecord.new
-      cache = ActiveSupport::Cache::MemoryStore.new
+      cache = Rails.cache
       # Flipper settings will be stored in postgres and cached in memory for 1 minute in production/staging
-      cached_adapter = Flipper::Adapters::ActiveSupportCacheStore.new(activerecord_adapter, cache, expires_in: 1.minute)
-      adapter = Rails.env.development? || Rails.env.test? ? activerecord_adapter : cached_adapter
-      instrumented = Flipper::Adapters::Instrumented.new(adapter, instrumenter: ActiveSupport::Notifications)
+      cached_adapter = Flipper::Adapters::ActiveSupportCacheStore.new(activerecord_adapter, cache,
```
Comment (f1337) [inline config/initializers/flipper.rb:19]: Why did we change the cache length to 5m?

File: config/initializers/flipper.rb [hunk 2]
```diff
@@ -14,11 +14,11 @@
...
+                                                                      expires_in: 5.minutes)
```
Comment (f1337) [inline config/initializers/flipper.rb:20]: Why did we change the cache expiration to 5m?

**Review by thilton-oddball — APPROVED** [review]
LGTM

**Pull request merged**